llvm/test/CodeGen/AMDGPU/fix-crash-valu-hazard.mir

```diff
@@ -0,0 +1,54 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -mcpu=gfx942 -verify-machineinstrs -run-pass post-RA-hazard-rec -o - %s | FileCheck %s
+
+--- |
+  @G = global <2 x i32> splat (i32 5)
+
+  define amdgpu_ps void @global_load_lds_dword_saddr(ptr addrspace(1) inreg nocapture %gptr, ptr addrspace(3) nocapture %lptr) #0 {
+  main_body:
+    %LGV = load <2 x i32>, ptr @G, align 8
+    %B = mul <2 x i32> %LGV, splat (i32 10)
+    call void @llvm.amdgcn.global.load.lds(ptr addrspace(1) %gptr, ptr addrspace(3) %lptr, i32 4, i32 32, i32 2)
+    store <2 x i32> %B, ptr @G, align 8
+    ret void
+  }
+
+  attributes #0 = { "amdgpu-memory-bound"="true" "amdgpu-wave-limiter"="true" "target-cpu"="gfx942" }
+
+...
```

Comment:
The point of the MIR test was to have a synthetic case with a minimal set of instructions, not a MIR copy of the original test. Can you add one to one of the existing hazard tests? e.g. hazards-gfx950.mir

Reply:
My bad, I'll do that

Reply:
Done, would appreciate it if you could check. Thanks !